{"name": "Отображается корректное сообщение об ошибке", "status": "passed", "start": 1734079735159, "stop": 1734079735259, "uuid": "63f4b750-47f9-4958-944f-7e68a998e789", "historyId": "41eb6f4645024213788f32ba69ce3881", "testCaseId": "41eb6f4645024213788f32ba69ce3881", "fullName": "tests.test_delete_courier.TestDeleteCourier#test_error_message_after_unsuccessful_request", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_delete_courier"}, {"name": "subSuite", "value": "TestDeleteCourier"}, {"name": "host", "value": "WIN-3SKQPE0FURM"}, {"name": "thread", "value": "10304-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_delete_courier"}], "links": [{"type": "link", "url": "https://qa-scooter.praktikum-services.ru", "name": "https://qa-scooter.praktikum-services.ru"}]}